app/assets/javascripts/search.coffee

```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```

Comment:
Rails magic is awesome, but it is nice to clean up all the autogenerated files you don't use whenever you get a chance (eg coffeescript files, empty helpers etc) :tada: